bench/protobuf.cc

```diff
@@ -0,0 +1,146 @@
+#include <benchmark/benchmark.h>
+
```

Comment:
let's put this into \tools folder please. Sounds good?

Reply:
Done! Clear to merge?